src/modules/dashboards/reducer.ts

```diff
@@ -79,16 +79,19 @@ const dashboardsReducer = (
         ...state,
         metadata: {
           ...state.metadata,
-          data: state.metadata.data.map((dashboardMeta) => {
-            if (action.payload.dashboardId === dashboardMeta.id) {
-              return {
-                ...dashboardMeta,
-                ...action.payload.metadata,
-              };
-            }
+          data: sortDashboards(
```

Comment:
Please, add additional test case that covers usage of `sortDashboards` in reducer.